README.md

```diff
@@ -1,184 +0,0 @@
-# otel-desktop-viewer
```

Comment:
I'm guessing you probably didn't want to delete this either?

Reply:
I wanted to rework it to reflect the changes with bringing in duckDb